Dockerfile

```diff
@@ -2,25 +2,21 @@ FROM node:14-alpine
 ENV IS_DOCKER true
 WORKDIR /app
 
-# Install system packages
```

Comment:
Please keep the comment.

Reply:
We should avoid useless comments, it's quite obvious that we're installing system packages, with `apt add`.

Do you see any case, where this comment could be useful ?
If you really want, I could add it again, sure, just wanted to know if there is a reason to keep it. :smile:

Reply:
Generally speaking I like to keep things divided into their own chunk with a short description of what is going next. Also, please keep in mind that not everyone is familiar with UNIX commands or whatsoever. So yes, please add them back.